### DISCUSSION TITLE
Locked Tabs in Editor Group

### DISCUSSION THREAD

**Discussion by yaq-cc** [coder/code-server#6376] [General]
Hi all,

I've bumped into this issue intermittently.  I'm running on Chrome Browser (latest).  Every once in a while, my tabs get "locked"; such that I can't move the tabs from editor group to editor group.  

I've tried looking this up a number of times; I'm convinced I've fat fingered my way here but I can't seem to find the right terms to Google up a fix.

how do "unlock" my editor tabs within my editor groups so I can move them around again?

Thank you!

**Comment by code-asher** [comment]
I see that there are [pinned tabs](https://code.visualstudio.com/updates/v1_46#_pin-tabs) and [editor group locks](https://code.visualstudio.com/updates/v1_60#_locked-editor-groups) but neither seem to prevent manually moving tabs from one editor group to another so I have no idea what it could be.
